src/ios/SimpleCameraPreview.m

```diff
@@ -358,4 +358,101 @@ - (void)deallocateMemory {
     locationManager = nil;
 }
 
+- (void) initVideoCallback:(CDVInvokedUrlCommand*)command {
+    self.videoCallbackContext = command;
+    NSDictionary *data = @{ @"videoCallbackInitialized" : @true };
+    
+    CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsDictionary:data];
+    [pluginResult setKeepCallbackAsBool:true];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+}
+
+- (void)startVideoCapture:(CDVInvokedUrlCommand*)command {
+    if (self.sessionManager != nil && !self.sessionManager.movieFileOutput.isRecording) {
+        NSArray *paths = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, YES);
+        NSString *libraryDirectory = [[paths objectAtIndex:0] stringByAppendingPathComponent:@"NoCloud"];
+        NSString* uniqueFileName = [NSString stringWithFormat:@"%@.mp4",[[NSUUID UUID] UUIDString]];
+        NSString *dataPath = [libraryDirectory stringByAppendingPathComponent:uniqueFileName];
+        NSURL *fileURL = [NSURL fileURLWithPath:dataPath];
+        [self.sessionManager startRecordingToOutputFileURL:fileURL recordingDelegate:self];
+    } else {
+       CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Session not initialized or already recording"];
+       [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
```

Comment:
setKeepCallback true

same in the other methods

Reply:
we keep it same logic as with Androids and the single callback